src/components/common/BuyCryptoButton/index.tsx

```diff
@@ -30,7 +31,7 @@ const useBuyCryptoHref = (): LinkProps['href'] | undefined => {
 }
 
 const buttonStyles = {
-  minHeight: '40px',
+  minHeight: '37.5px',
```

Comment:
What is this magic `37.5px` and why is it necessary in several places?

Reply:
@usame-algan ^^ :D ?

Reply:
If there is an outlined button and a contained button next to each other they have different heights because of the border. Also depending on the icon inside the button they can have different heights so setting the height this way was a quick solution. Imo we should fix this in our mui theme and align it with our style guide in Figma but I would do that separate from this PR.